examples/medplum-scheduling-demo/src/pages/UploadDataPage.tsx

```diff
@@ -90,6 +91,9 @@ async function uploadCoreData(medplum: MedplumClient): Promise<void> {
 }
 
 async function uploadExampleBots(medplum: MedplumClient, profile: Practitioner): Promise<void> {
+  // TODO: Friendlier error handling if the example bots json file hasn't been built yet
+  // stating that `npm run build:bots` should be run.
+  const exampleBotData = await import(EXAMPLE_BOTS_JSON);
```

Comment:
iirc you should be able to just `try.. catch` here and could display an error in the toast